packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.tsx

```diff
@@ -1724,9 +1745,15 @@ describe("MultiSelect", () => {
             await userEvent.click(textbox);
             await userEvent.paste("ear");
 
+            // wait to avoid getting caught in the Announcer debounce
+            jest.advanceTimersByTime(250);
```

Comment:
Is 250 a specific number that needs to be used? I see we also advance timers by 10!

I wonder if it would be helpful to add some docs around how to write unit tests for things using the announcer so other teams know how to create tests for it! Things like how to look for the announcer in the DOM, how long to advance timers, and any other internal details that would impact how it gets tested. Or maybe we could provide a testing utility to encapsulate these details? Just a thought!

Reply:
I love that idea! The `250` number is the default debounceThreshold. The 10 number came up when I was using a 0 value in tests... setTimeouts with a value of 0 weren't _quite_ immediate in the JavaScript event loop so I had to use 10! I can write a doc on testing this so the testing path is more well-trodden.

Reply:
Thinking about this more, do want consumers of the announcer to mock out the announcer instead so they don't have to worry about the implementation details? Their tests could be around making sure the announcer is called with the correct message! Then the announcer's unit tests are what makes sure it's putting the content in the live region correctly. What do you think?

Reply:
That's a great suggestion. I'd love to make the tests easier to work with, and mocking could be a really helpful way to do that!

Reply:
The Dropdown tests now use a mocked Announcer, so the internal details aren't important anymore. I can write some test how-tos in Confluence...that seemed like the best place for it!